db/seeds.rb

```diff
@@ -5,3 +5,14 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
+
+
+Books = ["Fellow", "1989", "Harry Potter", "Forever War", "Foundation", "Tubes"]
+Users = ["Kyle", "Jack", "Anna", "Kara", "Ben", "Jake"]
+
+
+
+Books.each do |book|
+  book = Book.create!(title: book)
+  puts "Created #{book.title}"
+end
```

Comment:
If you're building out functionality using tests, you shouldn't need a Seed file.